### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Samaamro20**
- Create a design for the Home page 
  - search bar 
  - about us section

### CODE REVIEW

File: client/src/components/Home/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import './home.css';
+
+export default class Home extends Component {
+
+  constructor(props) {
```
Comment (shahenazmonia) [inline client/src/components/Home/index.js]: You didn't pass any props until now! why did you identify props here?

File: package.json [hunk 2]
```diff
@@ -23,13 +23,17 @@
...
   "dependencies": {
+    "@blueprintjs/core": "^3.14.1",
     "env2": "^2.2.2",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.16.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-react": "^7.12.4",
     "express": "^4.16.4",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
```
Comment (shahenazmonia) [inline package.json]: why did you install ```path``` module? delete unrequired modules!

File: client/src/components/Home/index.js (hunk 1 above)
Comment (Samaamro20) [reply client/src/components/Home/index.js]: Done

File: package.json (hunk 2 above)
Comment (Samaamro20) [reply package.json]: Done

**Pull request merged**